src/components/victory-axis.jsx

```diff
@@ -507,22 +474,25 @@ export default class VictoryAxis extends React.Component {
       // Do less work by having `VictoryAnimation` tween only values that
       // make sense to tween. In the future, allow customization of animated
       // prop whitelist/blacklist?
-      const animateData = _.omit(this.props, [
-        "orientation", "scale", "tickFormat", "animate",
-        "crossAxis", "standalone"
+      const animateData = _.pick(this.props, [
+        "style", "domain", "range", "tickCount", "tickValues",
+        "labelPadding", "offsetX", "offsetY", "padding", "width", "height"
```

Comment:
Moving from `omit` to `pick` (the inverse) is a good idea in all our components, for animation performance. `pick` the smallest number of props, only the ones that make sense to animate.